test/test_pipeline/components/preprocessing/test_imputers.py

```diff
@@ -190,8 +183,7 @@ def test_constant_imputation(self):
             'X_train': data[train_indices],
             'dataset_properties': dataset_properties
         }
-        imputer_component = SimpleImputer(numerical_strategy='constant_zero',
-                                          categorical_strategy='constant_!missing!')
```

Comment:
Where do we test `constant_missing` now?

Reply:
we have removed it now, there is no categorical strategy for the simple imputer.

Reply:
Yeah, I know the background of this PR, but I mean we still have a categorical imputer somewhere, right?
Are we testing this imputer? This was my question.

Reply:
oh no sorry, that is being tested in feature validation.